src/circuit/gadgets/constraints.rs

```diff
@@ -430,6 +503,444 @@ pub fn must_be_simple_bit(x: &Boolean) -> AllocatedBit {
     }
 }
 
+pub fn enforce_comparison<F: LurkField, CS: ConstraintSystem<F>>(
```

Comment:
Please document what this function does and what its arguments represent.

Reply:
documented